### ISSUE TITLE
Multiple Graphite Servers

### ISSUE THREAD

**Issue by captncraig** [jwilder/gofana#2]
We currently use multiple graphite backends. Would it be possible to support multiple graphite hosts?

Something like `graphite-url http://aaaa:80,http://bbbb:8086`

**Comment by jwilder** [comment]
Yeah. That should be straightforward to support. Would you want to name them to distinguish them in the ui more easily?

**Comment by captncraig** [comment]
Perhaps that is getting complicated enough to not make sense on command line? 'http://foo:80;stats,http://bar:4444;somethingelse' could work though.
